components/StoryList/StyledStoryList.js

```diff
@@ -46,3 +64,49 @@ export const StoryCreationDate = styled.p`
   color: #666;
   margin: 0;
 `;
+export const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: inherit;
+`;
+// export const Button = styled.button`
+//   margin-bottom: 1rem;
+// `;
+export const AddStoryButton = styled.button`
+  align-self: center;
+  background-color: #fff;
+  background-image: none;
+  background-position: 0 90%;
+  background-repeat: repeat no-repeat;
+  background-size: 4px 3px;
+  border-radius: 15px 225px 255px 15px 15px 255px 225px 15px;
+  border-style: solid;
+  border-width: 2px;
+  box-shadow: rgba(0, 0, 0, 0.2) 15px 28px 25px -18px;
+  box-sizing: border-box;
+  color: #41403e;
+  cursor: pointer;
+  display: inline-block;
+  font-family: Neucha, sans-serif;
+  font-size: 1rem;
+  line-height: 23px;
+  outline: none;
+  padding: 0.75rem;
+  text-decoration: none;
+  transition: all 235ms ease-in-out;
+  border-bottom-left-radius: 15px 255px;
+  border-bottom-right-radius: 225px 15px;
+  border-top-left-radius: 255px 15px;
+  border-top-right-radius: 15px 225px;
```

Comment:
You are scaling a lot of your css properties with relative units and some with pixels. This is not a must do, but it might be better to use realtive units consistently, so your elements scale proportionally.

Reply:
Hey Thomas, 
first of all thx for your time. You are absolutely right . My styling is in a "experimental mode" right now 😅, tbh. For now i am focusing on the functionality of the App. Never the less i did start a little bit and i am going to try to fix these issues first before i continue😁

Reply:
Done.